artiq/examples/master/device_db.py

```diff
@@ -137,6 +137,13 @@
         "class": "SPIMaster",
         "arguments": {"channel": 23}
     },
+    
+    "spi_mmc": { 
+        "type": "local",
+        "module": "artiq.coredevice.spi",
+        "class": "SPIMaster",
+        "arguments": {"channel": 26}
+    },
```

Comment:
This change (and others) is already merged. Please don't submit them again (try ``git pull``).